helix-term/src/commands.rs

```diff
@@ -5234,7 +5234,8 @@ fn hover(cx: &mut Context) {
 
                 // skip if contents empty
 
-                let contents = ui::Markdown::new(contents, editor.syn_loader.clone());
+                let contents =
+                    ui::Markdown::new(contents, editor.syn_loader.clone()).style_group("hover");
```

Comment:
Change looks good, but do we need these extra scopes? We can simplify and remove `hover`/`completion` suffix

Reply:
`markup.raw.inline`， `markup.heading` not only used in editor veiw  but also used in popup ui. The two ui has own background. Some theme maybe conflict on these.